bees/jenkinsbee/jenkinsbee.go

```diff
@@ -80,8 +86,38 @@ func (mod *JenkinsBee) announceStatusChange(j Job) {
 	mod.eventChan <- event
 }
 
+func (mod *JenkinsBee) announceBuildStateChange(j Job) {
+	event := bees.Event{
+		Bee:  mod.Name(),
+		Name: "build_status_change",
+		Options: []bees.Placeholder{
+			{
+				Name:  "name",
+				Type:  "string",
+				Value: j.Name,
+			},
+			{
+				Name:  "url",
+				Type:  "string",
+				Value: j.URL,
+			},
+			{
+				Name:  "status",
+				Type:  "string",
+				Value: j.Color,
+			},
+			{
+				Name:  "building",
+				Type:  "string",
+				Value: strconv.FormatBool(j.LastBuild.Building),
+			},
+		},
+	}
+	mod.eventChan <- event
+}
+
 func (mod *JenkinsBee) poll() {
-	request, err := http.NewRequest("GET", mod.url+"/api/json", nil)
+	request, err := http.NewRequest("GET", mod.url+"/api/json?tree=jobs[name,url,color,lastBuild[building]]", nil)
```

Comment:
This is just filtering the output we want to retrieve, right? In other words: it will still report the data we need for the previously existing "status_change" event?